admin/CMakeLists.txt

```diff
@@ -31,9 +32,15 @@ string(REGEX REPLACE
     "${_herbert_on_template}"
 )
 file(WRITE "${LOCAL_INIT_DIR}/herbert_on.m" "${_herbert_on_script}")
+#
+if(${BUILD_TESTS})
+    configure_file("worker_4tests.m.template" "${LOCAL_INIT_DIR}/worker_4tests.m")
+endif()
+configure_file("worker_v2.m.template" "${LOCAL_INIT_DIR}/worker_v2.m")
+
 
 # Copy Herbert's worker template
-configure_file("${Herbert_WORKER_TEMPLATE}" "${LOCAL_INIT_DIR}/worker_v2.m")
+#configure_file("${Herbert_WORKER_TEMPLATE}" "${LOCAL_INIT_DIR}/worker_v2.m")
```

Comment:
this line and the associated comment can be deleted rather than commented out they're not being used.

Reply:
Also, if we're not using the `Herbert_WORKER_TEMPLATE` variable any more, we no longer need to find it in `FindHerbert`. So the block here: https://github.com/pace-neutrons/Horace/blob/c395b5344074c962b548bb114e565eb88c8bb990/cmake/FindHerbert.cmake#L56-L61 can be removed

Reply:
done